content/rule_book/internationals.md

```diff
@@ -33,7 +33,7 @@ These rules apply to the Internationals quiz meet conducted once yearly, adminis
 
 ### Scorekeeper
 
-- At least one scorekeeper will tabulate the results of each question on official score sheets in every International Final. They shall keep a running score of each quiz.
+- At least one scorekeeper will tabulate the results of each question on official scoresheets in every International Final. They shall keep a running score of each quiz.
```

Comment:
I don't care, but technically, [dictionary.com](https://www.dictionary.com/misspelling?term=scoresheet) doesn't think "scoresheet" is a word.

Reply:
I don't care either. Someone please choose between scoresheet/scoresheets and score sheet/score sheets.